src/Factory.php

```diff
@@ -26,4 +39,11 @@ public static function start(bool $isWriteable = false): void
             'read_and_close' => !$isWriteable,
         ]);
     }
+    private function get(string $key, string $default): string
+    {
+        if (!isset($this->environment[$key])) {
+            return $default;
+        }
+        return $this->environment[$key] ?: $default;
```

Comment:
is it intended that default is returned even when the environment is found?

Reply:
this returns the default if the environment is empty, yes, that is intended :)